http/src/request/channel/webhook/update_webhook.rs

```diff
@@ -131,3 +131,70 @@ impl TryIntoRequest for UpdateWebhook<'_> {
         Ok(request.build())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::error::Error;
+
+    #[test]
+    fn test_update_webhook() -> Result<(), Box<dyn Error>> {
+        const WEBHOOK_ID: Id<WebhookMarker> = Id::new(1);
+        const CHANNEL_ID: Id<ChannelMarker> = Id::new(2);
+
+        let client = Client::new("token".into());
+
+        {
+            let expected = r#"{}"#;
+            let actual = UpdateWebhook::new(&client, WEBHOOK_ID).try_into_request()?;
+
+            assert_eq!(Some(expected.as_bytes()), actual.body());
+        }
+
+        {
+            let expected = r#"{"avatar":"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI"}"#;
+            let actual = UpdateWebhook::new(&client, WEBHOOK_ID)
+            .avatar(Some("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI"))
+                .try_into_request()?;
+
+            assert_eq!(Some(expected.as_bytes()), actual.body());
+
+            let expected = r#"{"avatar":null}"#;
+            let actual = UpdateWebhook::new(&client, WEBHOOK_ID)
+                .avatar(None)
+                .try_into_request()?;
+
+            assert_eq!(Some(expected.as_bytes()), actual.body());
+        }
+
+        {
+            let expected = r#"{"channel_id":"2"}"#;
+            let actual = UpdateWebhook::new(&client, WEBHOOK_ID)
+                .channel_id(CHANNEL_ID)
+                .try_into_request()?;
+
+            assert_eq!(Some(expected.as_bytes()), actual.body());
+        }
+
+        {
+            let expected = r#"{"name":"Captain Hook"}"#;
+            let actual = UpdateWebhook::new(&client, WEBHOOK_ID)
+                .name(Some("Captain Hook"))?
+                .try_into_request()?;
+
+            assert_eq!(Some(expected.as_bytes()), actual.body());
+        }
```

Comment:
Let's also test a null name here like the null avatar

Reply:
Refer to #1748 where I remove the nullable name because this is not allowed.